README.md

```diff
@@ -60,39 +79,16 @@ A diversity of perspectives is important and anything you don't yet know, you ca
 
 ### Ethereum Classic Core Projects
 
-Ethereum Classic Clients public board https://trello.com/b/aiSzWBm7/core-etc-development
-
-Build Server (TeamCity) http://build.ethereumclassic.org/login.html
-
-Process of client restoration https://github.com/ethereumproject/volunteer/tree/master/Client%20Collaboration
-
 **Go-ethereum** 
 
-https://github.com/ethereumproject/go-ethereum
+https://github.com/ethereumclassic/go-ethereum
 
 Go language implementation of the original/classic design of the Ethereum protocol. Go-ethereum is currently restored and under active development, other clients are in the restoration process or require volunteers to contribute to their restoration. 
 
 Development builds https://bintray.com/ethereumproject/GoEthereumClassic/go-ethereum/develop#files/
 
 Contacts: IRC Server oftc.net, channel #etc-dev, contact @whatisgravity, Slack @splix, @elaine
 
-**Bomb Defusal**
-
-ECIP-1010 Delay Difficulty Bomb Explosion - https://github.com/ethereumproject/ECIPs/blob/master/ECIPs/ECIP-1010.md
-
-Related issues:
-
-https://github.com/ethereumproject/ECIPs/issues/4
-
-https://github.com/ethereumproject/volunteer/issues/28
-
-https://github.com/ethereumproject/go-ethereum/issues/11
-
-Public board https://trello.com/b/ZuTNRTsW/etc-bomb-disposal
-
-Contacts: IRC Server oftc.net, channel #etc-dev @whatisgravity, Slack @dontpanicburns, @splix, @elaine
-
-
 **Mist browser & wallet** 
 
 https://github.com/ethereumproject/mist
```

Comment:
We need to update endpoint later for this

Reply:
yes need to add emerald stuff as well